node/network/approval-distribution/src/lib.rs

```diff
@@ -1753,12 +2033,23 @@ impl ApprovalDistribution {
 			ApprovalDistributionMessage::NewBlocks(metas) => {
 				state.handle_new_blocks(ctx, metrics, metas, rng).await;
 			},
-			ApprovalDistributionMessage::DistributeAssignment(cert, candidate_index) => {
+			ApprovalDistributionMessage::DistributeAssignment(cert, candidate_indices) => {
+				// TODO: Fix warning: `Importing locally an already known assignment` for multiple candidate assignments.
```

Comment:
The approval subsystem warns with importing same assignment twice, since approval voting internally still tracks assignments/approvals individually, so when we `launch_approval` for as many times as bits in the core assignment bitfield. I have to either make this check here or in approval voting to not send multiple times, but the cost will be some extra memory and more code. Simplest hackier way would be to ignore this error for multiple core assignments, but I won't likely do it.

Reply:
Does this need to addressed before merge? If not, don't forget to create an issue :)

Reply:
This was done recently, but I did not delete this comment :)